contentcuration/contentcuration/tasks.py

```diff
@@ -221,6 +228,8 @@ def create_async_task(task_name, user, apply_async=True, **task_args):
     if "node_ids" in task_args:
         metadata["affects"]["nodes"] = task_args["node_ids"]
 
+    metadata['args'] = task_args
```

Comment:
Hoping this won't cause an issue, but I didn't want to have logic specifically for the diff task in a more general function